commands/azureCommands/create-registry.ts

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
Please add header to all new files (see #385):
/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Microsoft Corporation. All rights reserved.
 *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
 *--------------------------------------------------------------------------------------------*/